### DISCUSSION TITLE
阅读限制插件BUG

### DISCUSSION THREAD

**Discussion by wxyShine** [halo-dev/halo#6305] [Q&A]
### 系统信息

- 外部访问地址: https://www.wxy97.com/
- 启动时间: 2024-07-04 17:29
- 版本: 2.17.0
- 构建时间: 2024-07-02 10:37
- Git Commit: b1f23f8
- Java: IBM Semeru Runtime Open Edition / 21.0.3+9-LTS
- 数据库: PostgreSQL / 16.3 (Debian 16.3-1.pgdg120+1)
- 操作系统: Linux / 6.1.0-10-amd64
- 已激活主题: [Walker 1.8.1](https://www.halo.run/store/apps/app-GHgAR)
- 已启动插件:
  - [编辑器超链接卡片 1.0.3](https://github.com/halo-sigs/plugin-editor-hyperlink-card)
  - [页面静态缓存 1.0.0](https://github.com/halo-sigs/plugin-page-cache)
  - [Docsme 1.0.0-alpha.7](https://www.halo.run/store/apps/app-yffxw)
  - [豆瓣 1.1.9](https://github.com/chengzhongxue/plugin-douban)
  - [Data Studio（数据工厂） 1.0.0-alpha.2](https://github.com/halo-sigs/plugin-data-studio)
  - [友链自助提交插件 1.4.2](https://github.com/liuyiwuqing/halo-plugin-links-submit)
  - 联系表单 1.0.2
  - [Umami 1.4.0](https://github.com/halo-sigs/plugin-umami)
  - [lightgallery.js 灯箱 1.2.1](https://github.com/halo-sigs/plugin-lightgallery)
  - [highlight.js 代码高亮 1.2.2](https://github.com/halo-sigs/plugin-highlightjs)
  - [静态网页服务 1.0.0-alpha.4](https://www.halo.run/store/apps/app-gFkMn)
  - [OAuth2 认证 1.2.0](https://github.com/halo-sigs/plugin-oauth2)
  - [SEO 工具集 1.0.6](https://www.halo.run/store/apps/app-FNGbT)
  - [Vditor 编辑器 1.7.0](https://github.com/justice2001/halo-plugin-vditor)
  - [文本绘图 1.2.1](https://github.com/halo-sigs/plugin-text-diagram)
  - [Meilisearch 2.0.1](https://github.com/Rainsheep/halo-plugin-meilisearch)
  - [AI 助手 1.0.0](https://www.halo.run/store/apps/app-riNgb)
  - [增强备份（Backup IMproved） 1.1.1](https://www.halo.run/store/apps/app-dHakX)
  - [Metrics Graph 1.0.0](https://github.com/guqing/plugin-metrics-graph)
  - [Markdown / HTML 内容块 1.2.0](https://github.com/halo-dev/plugin-hybrid-edit-block)
  - [文章限制阅读 Pro 1.3.0](https://www.halo.run/store/apps/app-TzRqd)
  - [RSS 1.3.0](https://github.com/halo-dev/plugin-feed)
  - [Sitemap 1.1.2](https://github.com/halo-dev/plugin-sitemap)
  - [搜索组件 1.5.0](https://github.com/halo-dev/plugin-search-widget)
  - [评论组件 2.4.0](https://github.com/halo-dev/plugin-comment-widget)
  - [应用市场 1.2.0](https://www.halo.run/store/apps/app-VYJbF)
  - [瞬间 1.6.2](https://github.com/halo-sigs/plugin-moments)
  - [链接管理 1.4.0](https://github.com/halo-sigs/plugin-links)
  - [图库管理 1.3.0](https://github.com/halo-sigs/plugin-photos)
  - [对象存储（Amazon S3 协议） 1.10.0](https://github.com/halo-dev/plugin-s3)

### 使用的哪种方式运行？

Docker

### 发生了什么？

![d8bf896b1b8bc1f72950627c908cfa4](https://github.com/halo-dev/halo/assets/42802152/345abf9c-8dae-4337-aaab-31730043311d)
如图 
1.存在一条记录未展示出来
2. 无内容文案和上面不对照

**Comment by ruibaby** [comment]
感谢反馈此问题，但这个问题不属于此仓库的范围，我将转移到 Discussion，并和我们相关的插件开发人员反馈。

**Comment by Arch4AI** [comment]
已解决。
